src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-usersgroups-form/smb-usersgroups-form.component.ts

```diff
@@ -20,7 +20,7 @@ import { FinishedTask } from '~/app/shared/models/finished-task';
 import { TaskWrapperService } from '~/app/shared/services/task-wrapper.service';
 import { Group, SMBCluster, SMBUsersGroups, User, USERSGROUPS_RESOURCE } from '../smb.model';
 import { Location } from '@angular/common';
-import { USERSGROUPS_URL } from '../smb-usersgroups-list/smb-usersgroups-list.component';
+import { USERSGROUPS_PATH } from '../smb-usersgroups-list/smb-usersgroups-list.component';
```

Comment:
```suggestion
import { USERS_GROUPS_PATH } from '../smb-usersgroups-list/smb-usersgroups-list.component';
```
just for readability

Reply:
Did not change it to keep consistency with other references to `usersandgroups`

e.g https://docs.ceph.com/en/latest/mgr/smb/#users-and-groups-resource
https://github.com/ceph/ceph/tree/main/src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-usersgroups-list

Reply:
Thats the backend literal, we dont need to keep the same for our constants names which are for codebase maintenance. The reason you have a constant so you can maintain such backend literals properly.
This is a non-blocking comment, yet would be good if we adopt. Thanks